### PR TITLE
Refactor feature tests

### PR DESCRIPTION
**Pull request by bibicollins**
Added new feature tests to test more functionality. Also re-installed simplecov and simplecov-console, as we lost this during the revert to full rails stack, and it was not testing all of the correct files. All need to run bundle install after pulling this merge.

### CODE REVIEW

File: spec/features/user_can_log_in_spec.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'helpers/users_helper_spec'
+
+RSpec.feature 'Sign_in', type: :feature do
+  scenario 'Can Login after being signed up with correct email and password' do
+    sign_up_test
```
Comment (heuperman) [inline spec/features/user_can_log_in_spec.rb]: Rename helper to sign_up_new_user?

File: spec/features/user_can_log_in_spec.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    expect(page).to have_content('Welcome')
+    click_link 'Log out'
+    visit 'sign_in'
+    expect(page).to have_content('Log in')
```
Comment (heuperman) [inline spec/features/user_can_log_in_spec.rb]: Expectation not needed as you use them later on

File: spec/features/user_can_log_in_spec.rb (hunk 1 above)
Comment (bibicollins) [reply spec/features/user_can_log_in_spec.rb]: Yep done!

File: spec/features/user_can_log_in_spec.rb (hunk 2 above)
Comment (bibicollins) [reply spec/features/user_can_log_in_spec.rb]: Got rid of them now, used them mainly during writing to debug failing tests, but no longer necessary

**Pull request merged**